app/views/users/induction_coordinators/registrations/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l">Sign up</h2>
+
+    <%= form_for(resource, as: resource_name, url: induction_coordinator_registration_register_path) do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <%= f.govuk_email_field :email, autocomplete: "email", value: @email, readonly: true %>
+
+      <%= f.govuk_text_field :school_name, label: { text: "School name" }, value: @school.name, readonly: true %>
```

Comment:
If they have already confirmed the school, probably there's no need to show that on the form. Better to have a hidden field with the school_id that we can then later use to retrieve the school unambiguously without worrying about upcasing / downcasing or getting rid of spaces.

Reply:
There is also a hidden school ID 🙂